### ISSUE TITLE
Fix blocks

### ISSUE THREAD

**Issue by elad661** [sopel-irc/sopel#619]
The `blocks` command is extremely stupid and nobody knows how to use it properly, also the documentation is an outright joke and doesn't agree with reality. The command scheme is nonsensical.

This should be something that just works, not something you have to curse at. You should need to read the code to understand how to use this basic functionality.

**Comment by stopspazzing** [comment]
Its actually broken and doesn't work at all in sopel currently. another bug that needs to be fixed. The above should be marked as a tweak...

**Comment by xenithorb** [comment]
So uh.. how do I block people then? Is there a way to do that currently? Kinda need it

**Comment by maxpowa** [comment]
It does currently work. There was a bug when @stopspazzing posted that prevented it from working at all but it has since been fixed. See [this wiki page](https://github.com/sopel-irc/sopel/wiki/Making-Sopel-ignore-people) for more information on `.blocks`.

**Comment by Exirel** [comment]
It works or it doesn't work? That is the question, now that we are in 2019.

**Comment by dgw** [comment]
@Exirel relevant: #1355

tl;dr it kind of works, but (like everything else) the documentation lies :grin:

**Comment by Exirel** [comment]
Ok so I suggest to keep #1355 and to close this one! 💪

**Comment by dgw** [comment]
SGTM :sunglasses:
